### PR TITLE
Add final version of PCB part

### PR DESCRIPTION
**Pull request by EmileAlbert**

### CODE REVIEW

File: src/electronics/pcb/2_Starting_Board.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+Finally, the starter switch is plugged on the board. It consists of a relay, sending a binary information to the ROS operating system. Linked to a 0.5m
+cable, this system allows to launch the match sequence pulling on it. 
+
+### Getting Started
+
+The following code is launched with ROS (if you haven't read yet this section, it might be tricky) this is a simplified version of the code used in 
+2018 robots. The aim of this code is the setting of a parameter in a configuration file in the first place, next the code publish on the start topic 
+when the start switch is pulled. This last action indicates to all script listening this topic the beginning of a game. 
+
+'''python
```
Comment (bluebeel) [inline src/electronics/pcb/2_Starting_Board.md]: Not rendering the code use ```and not '''

File: src/electronics/pcb/2_Starting_Board.md [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+    startInterrupt = pi.callback(pin_start, pigpio.EITHER_EDGE, start)
+
+    update_team(None, pi.read(pin_team), None)
+else:
+    rospy.sleep(1)
+    update_team(None, None, None)
+
+while not rospy.is_shutdown():
+    pass
+'''
```
Comment (bluebeel) [inline src/electronics/pcb/2_Starting_Board.md]: too ``

File: src/electronics/pcb/2_Starting_Board.md [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+If you copied the code and connect all cable as the pictures shows, the pulling of the start cable induce a publication on the start ROS topic (visible 
+with an echo on this topic) and the modification of the switch place sets the team parameter ROS file. 
+
+The switch strategy and initialisation were planned to respectively set a different implemented strategy and launch a initialisation (tests) 
+sequence. It was not used in 2018 due to a lack of time but they are already on the PCB for a future robot.
+
+Finally, in the real implementation on ROS, the pin configuration is placed in a .yaml file gathering all information required about raspberry pinout.
+Then, the code used to get pin number is 
+
+'''yaml
```
Comment (bluebeel) [inline src/electronics/pcb/2_Starting_Board.md]: too ```

File: src/electronics/pcb/3_Motor_Board.md [hunk 4]
```diff
@@ -0,0 +1,239 @@
...
+
+The encoder sends the rotation information to the controller. In our code, we used the _Encoder.h_ library to handle it. At this moment, we have the 
+velocity of the wheel in rad/s.
+
+* Motor control
+
+The H bridge is used for power two motors at the same. One the left side of the board there are all control pins (PWM and DIR for the two motors)
+and power supply of command (5V). On the other side of the board, you have the motor output and the motor power supply (24V).
+
+'''cpp
```
Comment (bluebeel) [inline src/electronics/pcb/3_Motor_Board.md]: too ```

File: src/electronics/pcb/3_Motor_Board.md [hunk 5]
```diff
@@ -0,0 +1,239 @@
...
+
+The implemented PID regulator computes to reach the target velocity   
+So at this moment, you have an arduino UNO controlling motor in velocity thanks to a PID regulation. All regulation is visible in the Serial Window of
+the Arduino software. 
+
+Now the next step is to give the control of the motor velocity to ROS core on the raspberry. For that, we implemented a SPI connection with ROS. These 
+concepts are explained in the ROS serial section. On the motor side, a function is added on the main code on the arduino UNO to receive data from master
+and set value of PID (proportional gain, integrative gain, derivative gain and velocity setpoints.
+
+'''cpp
```
Comment (bluebeel) [inline src/electronics/pcb/3_Motor_Board.md]: too ```

**Review by bluebeel — CHANGES_REQUESTED** [review]
Code rendering

**Review by azerupi — APPROVED** [review]

**Pull request merged**